### PR TITLE
Wrap autofilled arguments with backticks

### PR DESCRIPTION
**Pull request by tgodzik**
Fixes https://github.com/scalameta/metals/issues/2926

### CODE REVIEW

File: mtags/src/main/scala-2/scala/meta/internal/pc/completions/ArgCompletions.scala [hunk 1]
```diff
@@ -140,7 +141,8 @@ trait ArgCompletions { this: MetalsGlobal =>
       params.flatMap { param =>
         val allMemebers = matchingTypesInScope(param.tpe)
         allMemebers.map { memberName =>
-          val editText = "" + param.name + " = " + memberName
+          val editText =
+            "" + Identifier.backtickWrap(param.name) + " = " + memberName
```
Comment (dos65) [inline mtags/src/main/scala-2/scala/meta/internal/pc/completions/ArgCompletions.scala:145]:
```suggestion
            Identifier.backtickWrap(param.name) + " = " + memberName
```

File: mtags/src/main/scala-2/scala/meta/internal/pc/completions/ArgCompletions.scala [hunk 2]
```diff
@@ -140,7 +141,8 @@ trait ArgCompletions { this: MetalsGlobal =>
       params.flatMap { param =>
         val allMemebers = matchingTypesInScope(param.tpe)
         allMemebers.map { memberName =>
```
Comment (dos65) [inline mtags/src/main/scala-2/scala/meta/internal/pc/completions/ArgCompletions.scala:143]: old typo:
```suggestion
        val allMembers = matchingTypesInScope(param.tpe)
        allMembers.map { memberName =>
```

**Review by dos65 — APPROVED** [review]
LGTM!

File: mtags/src/main/scala-2/scala/meta/internal/pc/completions/ArgCompletions.scala (hunk 1 above)
Comment (tgodzik) [reply mtags/src/main/scala-2/scala/meta/internal/pc/completions/ArgCompletions.scala:145]: Fixed!

File: mtags/src/main/scala-2/scala/meta/internal/pc/completions/ArgCompletions.scala (hunk 2 above)
Comment (tgodzik) [reply mtags/src/main/scala-2/scala/meta/internal/pc/completions/ArgCompletions.scala]: Done!

**Pull request merged**